### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by AlessioGr** [payloadcms/payload#531] [General]
Is there a discord server for Payload CMS? If not, would you be willing to create one? I think that would be great to chat & connect with other people using Payload.

**Comment by rjdjohnston** [comment]
I just asked this same question in a comment on another thread. Hopefully they'll make one and we can support each other. If not, maybe we start an unofficial payload community :)

**Reply by jmikrut** [reply]
Yep - we've actually gotten this question like 5 times this week already! We're going to open one up. We'll make an announcement when we do!

**Comment by DanRibbens** [comment]
Alright, it is happening! https://github.com/payloadcms/payload/pull/541

**Reply by rjdjohnston** [reply]
Tried the link, get invalid server. Maybe it’s not ready?

![CC90BD73-390A-4160-82D7-4FA42C909D4F](https://user-images.githubusercontent.com/293686/164954505-ef3d3f13-1b77-43c3-a5db-da8169531aa2.png)

**Reply by rjdjohnston** [reply]
This is what I used: https://discord.com/invite/agbr9KJz

**Reply by DanRibbens** [reply]
Sorry I didn't realize that link would expire. Here try this: https://discord.com/invite/r6sCXqVk3v

**Reply by balmacefa** [reply]
Hi guys, This link is not on the website:

![image](https://user-images.githubusercontent.com/8296124/184264325-bab5a975-2260-483e-b143-88ee62984973.png)
